src/utils.js

```diff
@@ -8,3 +8,8 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findAverage = (...numbers) => {
+ return numbers.reduce((a,b) => a + b, 0) / numbers.length;
```

Comment:
Lets round the number to the nearest 2 decimal places if it is a float

Reply:
Let's ensure the following for this function

- at least one number must be provided. if no numbers are provided (e.g findAverage(), an error should be thrown)
- if any of the provided arguments is not a number, you should throw an error

Reply:
noted